Itellij-workspace/Data Structures Through Java Lab Programs/Program1.java

```diff
@@ -6,24 +6,28 @@
 import static java.lang.Math.sqrt;
 public class Program1 {
 
-	public static void main(String[] args) {
-		Scanner s = new Scanner(System.in);
-		System.out.print("Enter a Value : ");
-		int a = s.nextInt();
-		System.out.print("Enter b Value : ");
-		int b = s.nextInt();
-		System.out.print("Enter c Value : ");
-		int c = s.nextInt();
-		if((b*b)-(4*a*c)<0)
-		{
-			System.out.print("There are No Real Solutions !!");
-		}
-		else
-		{
-			int r1 = (int)Math.round((((-b) +sqrt((b*b)-(4*a*c)) ) / 2));
-			int r2 = (int)Math.round((((-b) -sqrt((b*b)-(4*a*c)) ) / 2));
-			System.out.print("The roots of equation ("+a+")X^2 +("+b+")X +"+"("+c+") are "+r1+","+r2);
-		}
-	}
+    public static void main(String[] args) {
+        Scanner s = new Scanner(System.in);
+        System.out.println("Equation is:: ax^2 + bx + c");
+        System.out.print("Enter \"a\" Value : ");
+        double a = s.nextDouble();

```

Comment:
Taking doubles as inputs
Even if you type just '5' for a, it is converted to '5.0'!!